src/utils/hints/contractsHints.js

```diff
@@ -3,4 +3,18 @@ export const contractsHints = {
   contractCreated: 'Keyblock height and exact date and time when the smart contract was created.',
   contractTxHash: 'The transaction hash of a ContractCreateTx that was executed to create the smart contract on the blockchain.',
   contractCreator: 'Account that created the smart contract.',
+  bytecode: 'Low-level representation of the smart contract that is generated when the contract\'s source code is compiled.',
+  contractsAccount: 'The address where the smart contract is located. To be used if a smart contract is already deployed.',
+  contractsAccountBalance: 'Smart contract account balance that is needed to execute "payable" functions.',
+  contractCalls: 'Amount of calls to the smart contract.',
+  apiLinks: 'Links to sources of the smart contract data.',
+  callTransactionsHash: 'Low-level representation of the smart contract that is generated when the contract\'s source code is compiled.',
+  callTransactionsTime: 'Estimated date, time and block height when the smart contract was called.',
```

Comment:
```suggestion
  callTransactionsTime: 'Estimated date, time and keyblock height when the smart contract was called.',
```

is it estimated or exact date? also it seems like "estimated" also relates to the keyblock height which reads weird 🤔 please compare with other statements, I think we have multiple places for this

Reply:
you are right, the time is exact.
I rephrased the sentence the same as in previous case